app/models/description.rb

```diff
@@ -566,10 +522,61 @@ def update_users_and_parent
       SiteData.update_contribution(:del, editors_join_table, user.id)
     end
 
+    return unless parent.description_id == id
+
     # Make sure parent doesn't point to a nonexisting object.
-    if parent.description_id == id
-      parent.description_id = nil
-      parent.save_without_our_callbacks
+    parent.description_id = nil
+    parent.save_without_our_callbacks
+  end
+
+  ##############################################################################
+
+  private
+
+  # Descriptive subtitle for this description (when it is not necessary to
+  # include the title of the parent object), in plain text.  [I'm not sure
+  # I like this here.  It might violate MVC a bit too flagrantly... -JPH]
+  def put_together_name(full_or_part)
+    tag = :"description_#{full_or_part}_title_#{source_type}"
+    user_name = begin
+                  user.legal_name
+                rescue StandardError
+                  "?"
+                end
+    args = {
+      text: source_name,
+      user: user_name
+    }
+    if full_or_part == :full
+      args[:object] = parent.format_name
+    elsif source_name.present?
+      tag = :"#{tag}_with_text"
     end
+    tag.l(args)
+  end
+
+  # Change a given User's or UserGroup's privileges.
+  def chg_permission(groups, arg, mode)
+    arg = UserGroup.one_user(arg) if arg.is_a?(User)
+    if (mode == :add) &&
+       !groups.include?(arg)
+      groups.push(arg)
+    elsif (mode == :remove) &&
+          groups.include?(arg)
+      groups.delete(arg)
+    end
+  end
+
+  def user_made_a_change?(user)
+    self.class.connection.select_value(%(
+      SELECT id FROM #{versioned_table_name}
```

Comment:
#### Question: Is there a way to change this so that brakeman won't re-raise the complaint if this method is tweaked?
I could tell CodeClimate that this is a false positive.
brakeman is highlighting `versioned_table_name`, see attached jpg. That's not user input. 
![Screen Shot 2020-12-21 at 07 52 59](https://user-images.githubusercontent.com/45460/102796766-9aa10800-4363-11eb-87ca-723936634f8a.jpg)
But if the SQL gets changed, see, e.g., https://github.com/MushroomObserver/mushroom-observer/pull/728#discussion_r546760871, it would be nice if brakeman doesn't complain again. 
So can the SQL be changed to guarantee that brakeman won't complain in the future? 
What about a prepared statement (or is that not worth the effort)?

Reply:
I will use the simplest solution: tell CodeClimate this is a false positive.